config/jobs/kubernetes/sig-security/cve-feed-tests.yaml

```diff
@@ -13,11 +13,10 @@ presubmits:
     spec:
       containers:
       - image: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20250116-2a05ea7e3d
-        workingDir: sig-security-tooling/cve-feed/hack
         command:
         - python
         args:
-        - test_cve_title_parser.py
+        - sig-security-tooling/cve-feed/hack/test_cve_title_parser.py
```

Comment:
are we sure the working dir is going to be the `sig-security` clone? I guess yes

Reply:
I think so, anyway; all the working examples I looked at just ran files that are in the root of their repository